source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/IntegrationTests/EndpointTests/MeteringPointPersisterEndpointTests.cs

```diff
@@ -137,19 +135,16 @@ public static string RandomString(int length)
                     MeteringMethod = MeteringMethod.MmPhysical,
                     SettlementMethod = settlementMethod,
                 };
-                parentId = $"00-{correlationId}-b7ad6b7169203331-01";
 
                 var byteArray = message.ToByteArray();
-                var serviceBusMessage = new ServiceBusMessage(byteArray)
-                {
-                    CorrelationId = correlationId,
-                };
+                var serviceBusMessage = new ServiceBusMessage(byteArray) { CorrelationId = correlationId };
                 serviceBusMessage.ApplicationProperties.Add("OperationTimestamp", date.ToUniversalTime());
                 serviceBusMessage.ApplicationProperties.Add("OperationCorrelationId", "1bf1b76337f14b78badc248a3289d021");
```

Comment:
nit:  Not part of your change, but should the value have been CorrelationId here?
```suggestion
                serviceBusMessage.ApplicationProperties.Add("OperationCorrelationId", CorrelationId);
```

Reply:
Well spotted! I will fix that in pull request #1567